addon/adapters/json-api.js

```diff
@@ -99,7 +100,7 @@ var JSONAPIAdapter = RESTAdapter.extend({
     @return {Promise} promise
   */
   findMany(store, type, ids, snapshots) {
-    if (isEnabled('ds-improved-ajax')) {
+    if (isEnabled('ds-improved-ajax') && !this._hasCustomizedAjax()) {
```

Comment:
Ah, awesome. Sorry for making you check this, I could have easily done this myself 😔